.circleci/config.yml

```diff
@@ -155,14 +155,17 @@ commands:
       - run:
           name: Install asdf
           command: |
+            git config advice.detachedHead false
             if [ ! -d $HOME/.asdf ]; then
               git clone https://github.com/asdf-vm/asdf.git $HOME/.asdf --branch << parameters.asdf_version >>
               cd $HOME/.asdf
             else
               cd $HOME/.asdf
               git fetch origin
-              git pull
+              git fetch --all
             fi
+            git checkout master
+            git pull --all
```

Comment:
are these necessary?

Reply:
Yes. When pulling from asdf with a version, for some reason it puts you headless. I needed to checkout master, and then wanted to confirm everything was up to date (particularly just to ensure breaking caches).